examples/ht.py

```diff
@@ -50,9 +49,35 @@ def triangle_hits(o):
 hour_of_day = int(hh) + (int(mm) * (1 / 60))
 #print("hour_of_day: ", hour_of_day)
 
+#rotation
+theta = np.radians(30)
+# in normal coordinates; y here
+y = np.array(( (np.cos(theta), -np.sin(theta), 0),
+               (np.sin(theta), np.cos(theta), 0),
+               (0, 0, 1) ))
+x = np.array(( (1, 0, 0),
+               (0, np.cos(theta), -np.sin(theta)),
+               (0, np.sin(theta), np.cos(theta)) ))
+#y in normal coordinates; z here
+z = np.array(( (np.cos(theta), 0, np.sin(theta)),
```

Comment:
It's usually a bit safer to supply lists and lists of lists than tuples here.  So this would look like

`z = np.array( [ [ np.cos(theta), 0, np.sin(theta)],`